client-src/elements/form-field-specs.js

```diff
@@ -366,6 +367,26 @@ export const ALL_FIELDS = {
         public forum before sending an Intent to Prototype in
         order to enable discussion with other browser vendors,
         standards bodies, or other interested parties.`,
+    extra_help: html`
+        <p>
+        See the TAG guide to writing <a href="https://tag.w3.org/explainers/">Explainers</a>
+        for several examples of good explainers and tips for effective explainers.
+        </p>
+        <p>
+        You will need to publish this explainer by sharing it on a public forum,
+        and kick off "standards incubation" by proposing this to a relevant standards venue.
+        Frequently, this means you should
+        <a href="https://github.com/WICG/proposals#what-does-a-proposal-look-like">
+          make a WICG proposal</a> and start socializing
+        the problem with other vendors and developers.
```

Comment:
Since the "Initial Public Proposal" field sits before the "Explainer" field in the guide, maybe this should talk instead about "If you've already made an initial public proposal, post your explainer to that thread. Otherwise, your initial proposal can refer to you explainer." And then put the link to making a WICG proposal in the extra_help for the "Initial Public Proposal" field.